src/app/[locale]/incident/add/components/MapDialog.tsx

```diff
@@ -98,13 +140,15 @@ const MapDialog = ({ trigger }: MapDialogProps) => {
                   updateForm({ ...formState, coordinates: marker })
                 }
               >
-                <Button appearance="primary-action-button">Kies locatie</Button>
+                <Button appearance="primary-action-button">
```

Comment:
Willen we nog ergens laten zien welk adres geselecteerd is op de kaart? Of gaan we dat doen op een vergelijkbare manier zoals het nu geïmplementeerd is in signalen i.e. laten zien in het inputveld?

Reply:
Ik denk dat dat komt met de combobox #8. Denk dat het handig is om dan wel even te kijken wat het meest toegankelijk is. Ik denk dat het slim is om dit wel te bespreken :)